src/Database/LSMTree/Internal/RunAcc.hs

```diff
@@ -93,7 +93,7 @@ new :: NumEntries
                                   -- which is based on the 'NumPages'.
     -> ST s (RunAcc s)
 new (NumEntries nentries) npages rangeFinderPrecisionOverride = do
-    mbloom <- Bloom.Easy.easyNew 0.1 nentries -- TODO(optimise): tune bloom filter
+    mbloom <- Bloom.Easy.easyNew 0.02 nentries -- TODO(optimise): tune bloom filter
```

Comment:
This is in line with what the bloom filter macro-benchmark was using. It leads to a little over 9 bits per entry in the filter